### ISSUE TITLE
[bug] `yarn docs` command not compiling after clean install

### ISSUE THREAD

**Issue by aseeeem** [tldraw/tldraw#716]
attempted to run `yarn docs` after running `yarn install`

got the following error:

```
@tldraw/core:docs: Error: src/test/ContextWrapper.tsx:9:44 - error TS2339: Property 'children' does not exist on type '{}'.
@tldraw/core:docs: 
@tldraw/core:docs: 9 export const ContextWrapper: React.FC = ({ children }) => {
@tldraw/core:docs:                                              ~~~~~~~~
@tldraw/core:docs: 
@tldraw/core:docs: Error: src/test/renderWithContext.tsx:6:18 - error TS2559: Type '{ children: ReactNode; }' has no properties in common with type 'IntrinsicAttributes'.
@tldraw/core:docs: 
@tldraw/core:docs: 6   return render(<ContextWrapper>{children}</ContextWrapper>)
@tldraw/core:docs:                    ~~~~~~~~~~~~~~
@tldraw/core:docs: 
@tldraw/core:docs: Error: src/test/renderWithSvg.tsx:7:6 - error TS2559: Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes'.
@tldraw/core:docs: 
@tldraw/core:docs: 7     <ContextWrapper>
@tldraw/core:docs:        ~~~~~~~~~~~~~~
@tldraw/core:docs: 
@tldraw/core:docs: error Command failed with exit code 3.
@tldraw/core:docs: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
@tldraw/core:docs: Error: command finished with error: exit status 3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Node version: `16.14.2`
Yarn version: `1.22.19`

**Comment by steveruizok** [comment]
We should probably just pull `yarn docs` from the repo, I'm not sure how useful it is at this point.

**Comment by with-heart** [comment]
I believe this was resolved with #721. I did a fresh clone and can't reproduce using the same `node`/`yarn` versions as @aseeeem.

@steveruizok Yeah I've never really found type docs to be that useful. I'll open a PR for removing them

**Comment by aseeeem** [comment]
wowwww y’all are the best, thanks!
